src/routes/webhooks/apple.ts

```diff
@@ -0,0 +1,99 @@
+import { readFileSync } from 'node:fs';
+import { env } from 'node:process';
+import type { FastifyInstance, FastifyRequest } from 'fastify';
+import { logger } from '../../logger';
+import {
+  Environment,
+  SignedDataVerifier,
+} from '@apple/app-store-server-library';
+import { isTest } from '../../common';
+import { isInSubnet } from 'is-in-subnet';
+
+const loadAppleRootCAs = (): Buffer[] => {
+  if (isTest) {
+    return [readFileSync('__tests__/fixture/testCA.der')];
+  }
+
+  return [
+    readFileSync(
+      '/usr/local/share/ca-certificates/AppleIncRootCertificate.cer',
+    ),
+    readFileSync('/usr/local/share/ca-certificates/AppleRootCA-G2.cer'),
+    readFileSync('/usr/local/share/ca-certificates/AppleRootCA-G3.cer'),
+  ];
+};
+
+const getVerifierEnvironment = (): Environment => {
+  switch (env.NODE_ENV) {
+    case 'development':
+      return Environment.SANDBOX;
+    case 'test':
+      return Environment.LOCAL_TESTING;
+    default:
+      return Environment.PRODUCTION;
+  }
+};
+
+const bundleId = isTest ? 'com.example' : env.APPLE_APP_BUNDLE_ID;
```

Comment:
dev.fylla? 😄

Reply:
Copied from Apple's tests for the library, but I see I can change it if I use it the same in payload.